src/main/java/at/hannibal2/skyhanni/features/inventory/wardrobe/CustomWardrobe.kt

```diff
@@ -0,0 +1,570 @@
+        horizontalAlignment: HorizontalAlignment = HorizontalAlignment.CENTER,
+        verticalAlignment: VerticalAlignment = VerticalAlignment.CENTER,
+        hoveredColor: Color,
+        unHoveredColor: Color = hoveredColor,
+        borderOutlineThickness: Int,
+        borderOutlineBlur: Float = 0.5f,
+        onClick: () -> Unit,
+        onHover: () -> Unit = {},
+    ): Renderable =
+        Renderable.hoverable(
+            Renderable.drawInsideRoundedRectWithOutline(
+                Renderable.doubleLayered(
+                    Renderable.clickable(
+                        hoveredRenderable,
+                        onClick
+                    ), topLayerRenderable
+                ),
+                hoveredColor,
+                padding = padding,
+                topOutlineColor = config.color.topBorderColor.toChromaColorInt(),
+                bottomOutlineColor = config.color.bottomBorderColor.toChromaColorInt(),
+                borderOutlineThickness = borderOutlineThickness,
+                blur = borderOutlineBlur,
+                horizontalAlign = horizontalAlignment,
+                verticalAlign = verticalAlignment
+            ),
+            Renderable.drawInsideRoundedRect(
+                unhoveredRenderable,
+                unHoveredColor,
+                padding = padding,
+                horizontalAlign = horizontalAlignment,
+                verticalAlign = verticalAlignment
+            ),
+            onHover = { onHover() }
+        )
+
+    private fun WardrobeAPI.WardrobeSlot.clickSlot() {
+        val previousPageSlot = 45
+        val nextPageSlot = 53
+        val wardrobePage = WardrobeAPI.currentPage ?: return
+        if (isInCurrentPage()) {
+            if (isEmpty() || locked || waitingForInventoryUpdate) return
+            WardrobeAPI.currentSlot = if (isCurrentSlot()) null else id
+            InventoryUtils.clickSlot(inventorySlot)
+        } else {
+            if (page < wardrobePage) {
+                WardrobeAPI.currentPage = wardrobePage - 1
+                waitingForInventoryUpdate = true
+                InventoryUtils.clickSlot(previousPageSlot)
+            } else if (page > wardrobePage) {
+                WardrobeAPI.currentPage = wardrobePage + 1
+                waitingForInventoryUpdate = true
+                InventoryUtils.clickSlot(nextPageSlot)
+            }
+        }
+        update()
+    }
+
+    private fun WardrobeAPI.WardrobeSlot.getSlotColor(): Color = with(config.color) {
+        when {
+            isCurrentSlot() -> equippedColor
+            favorite -> favoriteColor
+            else -> null
+        }?.toChromaColor()?.transformIf({ isInCurrentPage() }) { darker() }
+            ?: (if (isInCurrentPage()) samePageColor else otherPageColor).toChromaColor()
+                .transformIf({ locked || isEmpty() }) { darker(0.2) }.addAlpha(100)
+    }
+
+    fun isEnabled() = LorenzUtils.inSkyBlock && WardrobeAPI.inWardrobe() && config.enabled
```

Comment:
Since the first review of this function is already outdated (changed line):
please use a cached result of inWardrobe check here. Also first check if the feature is enabled, and only then check if in wardrobe